packages/dashboard/src/util/theme.ts

```diff
@@ -0,0 +1,63 @@
+import { makeStyles } from '@material-ui/core';
+import { ThemeMode } from '../settings';
+
+const darkThemeStyles = makeStyles((theme) => ({
+  components: {
+    backgroundColor: theme.palette.primary.dark,
+    color: theme.fontColors.darkTheme,
+  },
+  background: {
+    backgroundColor: theme.palette.secondary.dark,
+  },
+  map: {
+    filter:
+      'invert(90%) sepia(12%) saturate(5773%) hue-rotate(193deg) brightness(92%) contrast(92%)',
```

Comment:
I guess these filters are based off the office map, can you test it with other maps in rmf_demos?